nixos/modules/services/mail/postfix.nix

```diff
@@ -719,45 +760,44 @@ in
             ExecStart = "${pkgs.postfix}/bin/postfix start";
             ExecStop = "${pkgs.postfix}/bin/postfix stop";
             ExecReload = "${pkgs.postfix}/bin/postfix reload";
+
+            StateDirectory = "postfix";
+            StateDirectoryMode = "0755";
+            ReadWritePaths = [ "/var/spool/mail" ];
+
+            CapabilityBoundingSet = "CAP_NET_BIND_SERVICE CAP_DAC_READ_SEARCH CAP_SETUID CAP_SETGID";
+            DevicePolicy = "closed";
+            LockPersonality = true;
+            MemoryDenyWriteExecute = true;
+            NoNewPrivileges = true;
+            PrivateDevices = true;
+            PrivateMounts = true;
+            PrivateTmp = true;
+            ProtectClock = true;
+            ProtectControlGroups = true;
+            ProtectHome = true;
```

Comment:
Won’t this break .forwarding, or delivering mail to a local user’s home directory?

Reply:
Yes. Replacing `ProtectHome=true` by `ReadOnlyPaths=/home` would allow to read .forward. For delivery to the home directory this has to be disabled entirely.

Should this be behind an option? If yes, how whould an option look like?

Reply:
I would suggest you don't put this behind an option - don't include it at all. It isn't a reasonable default for a mail server. Add documentation so users who can and wish to will know how to add it.